BUILDING.md

```diff
@@ -34,14 +34,7 @@ $ ./configure
 $ make
 ```
 
-If your Python binary is in a non-standard location or has a
-non-standard name, run the following instead:
-
-```console
-$ export PYTHON=/path/to/python
-$ $PYTHON ./configure
-$ make
-```
+Note that the above requires that `python` resolve to Python 2.6/2.7 and not a newer version.
```

Comment:
Landing, but doing the `2.6/2.7` -> `2.6 or 2.7` per @thefourtheye's suggestion.